lib/types.js

```diff
@@ -98,10 +98,25 @@
  * @property {String[]} [forks]
  */
 
-/**
+/*
  * @typedef {Object.<string, any>} GeneralObject
  */
 
+/**
+  * @typedef {GeneralObject} CoreIdRecord
+  * @property {CoreId} coreId
+  * @property {StoreNamespace} namespace
+  * @property {IdentityId} identityId
+ */
+
+/**
+  * @typedef {GeneralObject} CoreIdRecordAggregate
+  * @property {CoreId[]} coreIds
```

Comment:
In what scenario would be have more than one coreId for a given namespace and identityId? Would that not be an error?